src/WebJobs.Script.WebHost/Diagnostics/LinuxAppServiceFileLogger.cs

```diff
@@ -46,7 +48,7 @@ public LinuxAppServiceFileLogger(string logFileName, string logFileDirectory, IF
         public int MaxFileSizeMb { get; set; } = 10;
 
         // Maximum time between successive flushes (seconds)
-        public int FlushFrequencySeconds { get; set; } = 30;
+         public int MaxFlushFrequencySeconds { get; set; } = 30;
```

Comment:
nit: looks like you've got an extra space here

Reply:
Thank you for calling this out. I have taken a look through the changed files for leading or trailing spaces.